test/e2e/framework/util.go

```diff
@@ -3089,10 +3097,10 @@ func WaitForPartialEvents(c clientset.Interface, ns string, objOrRef runtime.Obj
 	})
 }
 
-type updateDSFunc func(*extensions.DaemonSet)
+type updateDSFunc func(*apps.DaemonSet)
 
-func UpdateDaemonSetWithRetries(c clientset.Interface, namespace, name string, applyUpdate updateDSFunc) (ds *extensions.DaemonSet, err error) {
-	daemonsets := c.ExtensionsV1beta1().DaemonSets(namespace)
+func UpdateDaemonSetWithRetries(c clientset.Interface, namespace, name string, applyUpdate updateDSFunc) (ds *apps.DaemonSet, err error) {
+	daemonsets := c.AppsV1beta2().DaemonSets(namespace)
```

Comment:
AFAIK, for replicaset e2e tests, Ken suggested to leave them untouched.

Reply:
I already discussed this with @kow3ns, since we aren't going to run 1.8 e2e tests against a 1.7 cluster, this is fine. I see no harm updating this.

Reply:
Thanks for verifying it.